Makefile

```diff
@@ -247,21 +247,25 @@ swagger-docs: ## preview the API documentation
 
 .PHONY: buildx
 ifeq ($(BUILDX), bundles/buildx)
-buildx: bundles/buildx # build buildx cli tool
-else
-buildx:
+buildx: bundles/buildx ## build buildx cli tool
```

Comment:
In https://github.com/moby/moby/pull/40091 buildx is a required pre-req for all builds.
What happens if we don't define `buildx:`?

Reply:
It should be covered by the `.PHONY buildx`

```bash
mkdir foo && cd foo

echo ".PHONY: buildx" > Makefile

make buildx
# make: Nothing to be done for `buildx'.

echo $?
# 0

make nonexistingtarget
# make: *** No rule to make target `nonexistingtarget'.  Stop.

echo $?
2
```